laterpay/views/backend/pricing.php

```diff
@@ -66,14 +66,14 @@ class="lp_plugin-mode-indicator"
                                     if ( in_array( $slug, $hidden_post_types, true ) ) {
                                         continue;
                                     }
-                                    echo '<li><label title="' . esc_attr( $post_type_data->labels->name ) . '">';
-                                    echo '<input type="checkbox" name="laterpay_enabled_post_types[]" value="' . esc_attr( $slug ) . '" ';
-                                    if ( is_array( $enabled_post_types ) && in_array( $slug, $enabled_post_types, true ) ) {
-                                        echo 'checked';
-                                    }
-                                    echo '>';
-                                    echo '<span>' . esc_html( $post_type_data->labels->name ) . '</span>';
-                                    echo '</label></li>';
+                                    ?>
+                                    <li>
+                                        <label title="<?php echo esc_attr( $post_type_data->labels->name ); ?>">
+                                            <input type="checkbox" name="laterpay_enabled_post_types[]" value="<?php echo esc_attr( $slug ); ?>" <?php checked( is_array( $enabled_post_types ) ? in_array( $slug, $enabled_post_types, true ) : '' ); ?>>
```

Comment:
Should it be ? `checked( ( is_array( $enabled_post_types ) && in_array( $slug, $enabled_post_types, true ) );`

Alternatively you can put condition in variable and use it. @thrijith